lib/Backends/CPU/libjit/libjit_conv.cpp

```diff
@@ -113,21 +113,23 @@ void libjit_convDKKC8_foreach_xy_filter_pixels(
     unsigned depthStrips, unsigned sizeGroupY, size_t numChannels, float *outW,
     const float *inW, const float *filterW, const float *biasW,
     const size_t *outWdims, const size_t *inWdims, const size_t *filterWdims,
-    const size_t *biasWdims, size_t filterSize, size_t stride, size_t pad,
+    const size_t *biasWdims, size_t filterSize, size_t stride, size_t *pads,
     size_t group) {
   // The loops below look scary but the the idea is simple. We iterate over
   // the pixels in the output tensor and calculate the coordinate of the source
   // tensor. When we process the Y row we try to process [sizeGroupY] elements
   // at once. After we finish the row we handle the odd cases by handling one y
   // value at a time.
 
+  size_t pad_t = pads[0];
```

Comment:
I don’t think that this is correct. You are ignoring two of the pad dims.

Reply:
From my understanding, this pad_t and pad_l are used here to control the starting position. Since the output dims are calculated already from all of the pads, so no need to use pad_b and pad_r here.

Reply:
Got it. Let's add a comment that explains this.

Reply:
I added a comment in libjit_convolution_f function since the way of using pads are similar for different modes.